test/functional/test_framework/test_framework.py

```
@@ -1494,6 +1476,13 @@ def connect_nodes(self, a, b, *, peer_advertises_v2=None):
             if mn2.nodeIdx is not None:
                 mn2.get_node(self).setmnthreadactive(True)
 
+    def append_dip3_config(self, datadir):
+            append_config(datadir, [
```

Flake8 (7.3.0)

[error] over-indented

(E117)